src/services/orderbook_service.ts

```diff
@@ -157,20 +157,22 @@ export class OrderBookService {
         const paginatedApiOrders = paginationUtils.paginate(apiOrders, page, perPage);
         return paginatedApiOrders;
     }
-    constructor(meshClient: WSClient) {
+    constructor(meshClient?: WSClient) {
```

Comment:
is the `OrderBookService` able to function without a `meshClient` ?

Reply:
It can read, but it cannot write. Perhaps I should 500 on that POST then.

Reply:
I ended up implementing this.